core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1276,11 +1276,13 @@ private[spark] object Utils
       case t: Throwable =>
         val currentThreadName = Thread.currentThread().getName
         if (sc != null) {
-          logError(s"uncaught error in thread $currentThreadName, stopping SparkContext", t)
+          logError(
+            log"uncaught error in thread ${MDC(THREAD, currentThreadName)}, stopping SparkContext",
```

Comment:
Will we have `THREAD_ID` in the future? `THREAD_NAME`?

Reply:
Good point, I will update this.